indico/modules/events/abstracts/controllers/reviewing.py

```diff
@@ -183,10 +185,44 @@ class RHSubmitAbstractComment(RHAbstractBase):
     def _check_abstract_protection(self):
         return self.abstract.can_comment(session.user)
 
+    def _get_recipients(self):
+        judges = set()
+        conveners = set()
+        reviewers = set()
+
+        for entry in self.abstract.event.acl_entries:
```

Comment:
Not sure if this could be a problem performance-wise.. we loop over all the acl entries in the event here

Reply:
This should be fine... event ACLs are usually not that huge